app/src/main/java/divyansh/tech/animeclassroom/favorites/DynamicTabFragment.kt

```diff
@@ -0,0 +1,21 @@
+package divyansh.tech.animeclassroom.favorites
+
+import androidx.core.os.bundleOf
+import androidx.fragment.app.Fragment
+import dagger.hilt.android.AndroidEntryPoint
+import divyansh.tech.animeclassroom.R
+
+
+@AndroidEntryPoint
+class DynamicTabFragment : Fragment(R.layout.fragment_dynamic_tab) {
```

Comment:
@krishnachaitanya0107 Make a new package inside the favorites package, named screens. Put this dynamic Tab Fragment there

Reply:
Also, add data binding layout for this tab, and include the _binding and binding variables inside of the fragment, like in favorites fragment and add onCreateView() method call